3d_armor/api.lua

```diff
@@ -56,6 +56,10 @@ armor = {
 		steel = "default:steel_ingot",
 		fine_steel = "default:fine_steel_ingot"
 	},
+        materials_fabric= {
```

Comment:
`materials_advanced` was already determined to be a bit of a mistake, something being described as a "fabric" should be in the item definition's groups, not in yet another edge-case material table.

Reply:
I'm fine keeping this but @The32bitguy is going to double check the usage of `materials_advanced` elsewhere so we don't miss anything.